jira.go

```diff
@@ -31,6 +31,7 @@ type Field struct {
 	Assignees   *Assignee     `json:"assignee,omitempty"`
 	Priority    *PriorityType `json:"priority,omitempty"`
 	Labels      []string      `json:"labels,omitempty"`
+	DueDate     string        `json:"dueDate,omitempty"`
```

Comment:
This should be =>  DueDate     string        `json:"due**d**ate,omitempty"`  D small letter

Reply:
It seems to be different for you. I've been running like this for months and it matched our Jira Cloud due date field.